x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -220,379 +198,410 @@ export function App({
       }
     });
   }, [
-    lastKnownDoc,
     onAppLeave,
-    state.persistedDoc,
+    lastKnownDoc,
     state.isSaveable,
-    core.application.capabilities.visualize.save,
+    state.persistedDoc,
     getLastKnownDocWithoutPinnedFilters,
+    application.capabilities.visualize.save,
   ]);
 
   // Sync Kibana breadcrumbs any time the saved document's title changes
   useEffect(() => {
-    core.chrome.setBreadcrumbs([
-      ...(originatingApp && getAppNameFromId
-        ? [
-            {
-              onClick: (e) => {
-                core.application.navigateToApp(originatingApp);
-              },
-              text: getAppNameFromId(originatingApp),
-            } as EuiBreadcrumb,
-          ]
-        : []),
-      {
-        href: core.http.basePath.prepend(`/app/visualize#/`),
+    const byValueMode =
```

Comment:
Because it's a boolean, I recommend changing the name:

```suggestion
    const isByValueMode =
```

Reply:
Renamed this